### ISSUE TITLE
Any chance of pulling placeholder out of VariableEditor into a prop?

### ISSUE THREAD

**Issue by Nathansbud** [mac-s-g/react-json-view#333]
"Update this value" is pretty good, but doesn't always fit the bill!

**Comment by mac-s-g** [comment]
good point.  i'm conscious of prop bloat.  there's already quite a bit going on.
maybe we can separate out props related to editing into it's own object.
something like this:
```
editor: {
   onEdit: () => {},
   validationMessage: "...",
   placeholder: "..."   ​
}
```

just a thought.  it's not perfect so I'm open to ideas.
